docs/source/demos/lead_scoring.ipynb

```diff
@@ -286,12 +293,19 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "When we optimize for AUC, we can see that the AUC score from this pipeline is better than the AUC score from the pipeline optimized for lead scoring. However, the revenue per lead gained was only `$7` per lead when optimized for AUC and was `$45` when optimized for lead scoring. As a result, we would gain up to 6x the amount of revenue if we optimized for lead scoring.\n",
+    "When we optimize for AUC, we can see that the AUC score from this pipeline is better than the AUC score from the pipeline optimized for lead scoring. However, the revenue per lead suffered a loss of `$0.03` per lead when optimized for AUC and was `$14.6` when optimized for lead scoring. As a result, we would have a huge gain on the amount of revenue if we optimized for lead scoring.\n",
```

Comment:
This change in performance is interesting. I think it can partially be attributed to the data splitter (although I received similar results when reverting to StratifiedKFold). However, if we compare the pipelines in the [current version](https://feature-labs-inc-evalml--2139.com.readthedocs.build/en/2139/demos/lead_scoring.html) to the previous pipelines in a version that performed [better](https://evalml.alteryx.com/en/v0.15.0/demos/lead_scoring.html#Describe-pipeline), we see that we switched from using text featurizer to using OHE, which could explain some of the change.

Reply:
Hmmm... what should the job column be? Text or categorical? If the change in performance is because of how we represent and thus process data, maybe the solution should be to use the text representation if that makes sense for the job col? Taking a quick peek, it looks like if we represent it as a categorical col then we'll end up dropping a lot of potential info since each category doesn't appear very often.

Reply:
@angela97lin I tried changing it to text, but with the new limitations of not accepting NaN values in Natural Language columns, we would either need to manually impute beforehand or drop all the NaN columns